tests/server/core/error-pages.js

```diff
@@ -129,12 +129,37 @@ describe('Templating error pages', function () {
 			retval = ret;
 		});
 
-		renderer.render.firstCall.yield(new Error('bad times'));
+		renderer.render.firstCall.yield(new Error('renderer.render threw some error'));
 
 		assert.strictEqual(renderer.render.callCount, 1);
 		assert.strictEqual(null, retval);
 	});
 
+	it('Should callback with null if error falsy', function () {
+		var errorPages = require(moduleName)(config);
+		var retval = false;
+		var error;
+		errorPages.getPage(error, req, res, function (ret) {
+			retval = ret;
+		});
+
+		assert.strictEqual(null, retval);
+	});
+
+	it('Should render if provided err.status falsy', function () {
+		var errorPages = require(moduleName)(config);
+		var retval = false;
+		error.status = undefined;
+		errorPages.getPage(error, req, res, function (ret) {
+			retval = ret;
+		});
+
+		renderer.render.firstCall.yield(null, 'my error page');
+
+		assert.strictEqual(renderer.render.callCount, 1);
```

Comment:
I'd agree but there is a reason it's that way round, and it's like that throughout the tests... I think in the event of some of the test code failing it provides more useful error messages... @moddular ?